tests/tasks/assert_certificate_parameters.yml

```diff
@@ -12,7 +12,7 @@
   pip:
     name: certreader>=0.1.1
     virtualenv: "{{ __virtualenv_path }}"
-    virtualenv_command: python3 -m venv
+    virtualenv_command: /usr/bin/python3 -m venv
```

Comment:
?

Reply:
This was required because my tests where failing with: 
`FAILED! => {"changed": false, "msg": "Failed to find required executable python3 -m venv in paths: /sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin"}`

I've added on same commit and add a note in the commit message. Do you think it's better to add an extra commit for that?

Reply:
I guess it is ok to leave it in, as it appears to be related to making certitificate work on ansible 2.8

But - do you have to specify `virtualenv_command`?  The `pip` module isn't smart enough to know to use `virtualenv` with python 2 and `python -mvenv` with python3?

Reply:
in ansible 2.8 the `pip` module expects `virtualenv_command` to be a single string: https://github.com/ansible/ansible/blob/stable-2.8/lib/ansible/modules/packaging/language/pip.py#L457 - so I'm not sure how it works with `/usr/bin/python3 -m venv`

Reply:
I see - it works by accident - because `os.path.basename("/usr/bin/python3 -m venv")` is `python3 -m venv` which is != `/usr/bin/python3 -m venv` . . .